metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -543,37 +556,42 @@ object UserConfiguration {
 
     val customProjectRoot = getStringKey("custom-project-root")
 
+    val telemetryLevel = getStringKey("telemetry-level")
+      .flatMap(TelemetryLevel.fromString)
+      .getOrElse(TelemetryLevel.default)
+
     if (errors.isEmpty) {
       Right(
         UserConfiguration(
-          javaHome,
-          sbtScript,
-          gradleScript,
-          mavenScript,
-          millScript,
-          scalafmtConfigPath,
-          scalafixConfigPath,
-          symbolPrefixes,
-          worksheetScreenWidth,
-          worksheetCancelTimeout,
-          bloopSbtAlreadyInstalled,
-          bloopVersion,
-          bloopJvmProperties,
-          ammoniteProperties,
-          superMethodLensesEnabled,
-          showInferredType,
-          showImplicitArguments,
-          showImplicitConversionsAndClasses,
-          remoteLanguageServer,
-          enableStripMarginOnTypeFormatting,
-          enableIndentOnPaste,
-          enableSemanticHighlighting,
-          excludedPackages,
-          defaultScalaVersion,
-          disableTestCodeLenses,
-          javaFormatConfig,
-          scalafixRulesDependencies,
-          customProjectRoot,
+          javaHome = javaHome,
+          sbtScript = sbtScript,
+          gradleScript = gradleScript,
+          mavenScript = mavenScript,
+          millScript = millScript,
+          scalafmtConfigPath = scalafmtConfigPath,
+          scalafixConfigPath = scalafixConfigPath,
+          symbolPrefixes = symbolPrefixes,
+          worksheetScreenWidth = worksheetScreenWidth,
+          worksheetCancelTimeout = worksheetCancelTimeout,
+          bloopSbtAlreadyInstalled = bloopSbtAlreadyInstalled,
+          bloopVersion = bloopVersion,
+          bloopJvmProperties = bloopJvmProperties,
+          ammoniteJvmProperties = ammoniteProperties,
+          superMethodLensesEnabled = superMethodLensesEnabled,
+          showInferredType = showInferredType,
+          showImplicitArguments = showImplicitArguments,
+          showImplicitConversionsAndClasses = showImplicitConversionsAndClasses,
+          remoteLanguageServer = remoteLanguageServer,
+          enableStripMarginOnTypeFormatting = enableStripMarginOnTypeFormatting,
+          enableIndentOnPaste = enableIndentOnPaste,
+          enableSemanticHighlighting = enableSemanticHighlighting,
+          excludedPackages = excludedPackages,
+          fallbackScalaVersion = defaultScalaVersion,
+          testUserInterface = disableTestCodeLenses,
+          javaFormatConfig = javaFormatConfig,
+          scalafixRulesDependencies = scalafixRulesDependencies,
+          customProjectRoot = customProjectRoot,
+          telemetryLevel = telemetryLevel,
```

Comment:
Tha'ts the reason why I've follow the boy scout rule and added them. With so many unnamed parameters it's really easy to mix stuff up, especially, since lots of parameters have the same type (`Option[String]`, `Boolean`, etc). I'll add the scalaCliLauncher entry